index.js

```diff
@@ -18,59 +18,105 @@ const {
   writeJsonFile,
   dirFileExists,
   isEmptyDir,
-  getMostRecentDirectory
+  getMostRecentDirectory,
+  removeDir,
+  renameSync,
+  deleteFile
 } = require('./utils/fileDirOps');
 const { installPackages } = require('./utils/install');
 const { setupTurboRepoProject } = require('./utils/turboRepoSetup');
+const util = require('util');
+
+const UV2 = 'universal-react-v2';
```

Comment:
V2 is a temporary name we have created for time being, its going to be just universal-react or create-universal-react